app/forms/state_file/nj_college_dependents_exemption_form.rb

```diff
@@ -0,0 +1,17 @@
+module StateFile
+  class NjCollegeDependentsExemptionForm < QuestionsForm
+    attr_accessor :dependents_attributes
+    delegate :dependents, to: :intake
+
+    set_attributes_for(:intake)
+
+    def save
+      @intake.update!(attributes_to_save)
+    end
+
+    def attributes_to_save
+      base_attrs = attributes_for(:intake)
+      base_attrs.merge({ dependents_attributes: dependents_attributes.to_h })
```

Comment:
[sand] Does `base_attrs` contain anything? I think it might be an empty hash. If so, you may be able to just return `{dependent_attributes: dependent_attributes.to_h}`

Reply:
You're right, this can be removed. Will fix